contrib/opencensus-context/examples/py37.py

```diff
@@ -0,0 +1,51 @@
+# Copyright 2019, OpenCensus Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+
+from opencensus.common.runtime_context import RuntimeContext
+
+RuntimeContext.register_slot('correlation_context', lambda: dict())
+
+
+async def hello(name):
+    correlation_context = RuntimeContext.correlation_context.copy()
```

Comment:
I think register slot should return a var, and the current value can be retrieve only if you have access to that var similar with contextvars.

Reply:
Just added, that's also suggested by @c24t. Added the return value.

Regarding "and the current value can be retrieve only if you have access to that var", what about explicit thread propagation - for example, I want to create an explicit thread and pass **all the current context**, I don't know how many context I have. Currently I'm doing this in RuntimeContext.with_current_context, would like to get your opinion.